src/components/TransactionItemComponent/DataCeils/MerchantCell.tsx

```diff
@@ -0,0 +1,21 @@
+import useThemeStyles from '@hooks/useThemeStyles';
+import TextWithTooltip from '@components/TextWithTooltip';
+import React from 'react';
+import colors from '@styles/theme/colors';
+import type DataCellProps from './DateCellProps';
+
+function MerchantCell({transactionItem, showTooltip, isLargeScreenWidth}: DataCellProps) {
```

Comment:
this is a bit different than `MerchantCell` in TransactionRowItem - is there any reason? would be best if we could later replace the old components with this ones

Reply:
The components here differ subtly from those in `TransactionRowItem` because they are adapted to the onyx transaction and those in the TransactionRowItem are adapted to `TransactionListItemType` witch is more search specific type.

Reply:
yes I understand, but we are very close to be able to use this component both in the new reports code and in Search, so I would like to try to make them be usable also in Search 🤔 
But perhaps that could be worked on in a different PR